install/installer/pkg/config/versions/versions.go

```diff
@@ -29,7 +29,6 @@ type Components struct {
 	} `json:"imageBuilderMk3"`
 	InstallationTelemetry Versioned `json:"installationTelemetry"`
 	IntegrationTests      Versioned `json:"integrationTests"`
-	Kedge                 Versioned `json:"kedge"`
```

Comment:
> got old versions of config that do define this field

Two comments on this:
 - this is the versions file. We don't use it as config, plus no-one _should_ do so
 - if this were a relevant config field: instead of manually managing obsolescence of single fields, which involves multiple PRs, managing phase-out periods and such, we should instead make our parsing behavior in installer less picky (e.g., with a "warn but continue anyway" flag). The current hard lock-in between a) installer binary, b) versions file and c) config is not flexible enough for RL usecases.

We should be cautious where it make sense, but also watch out for our pace. :slightly_smiling_face: